src/04_printing.py

```diff
@@ -11,7 +11,8 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
-
+print("x is %d, y is %f, z is %s" % (x, y, z))
```

Comment:
it is possible to round the printed version of y using some syntax, try searching "python format print float"